pkg/system/chtimes_windows.go

```diff
@@ -24,3 +24,9 @@ func setCTime(path string, ctime time.Time) error {
 	c := windows.NsecToFiletime(windows.TimespecToNsec(ctimespec))
 	return windows.SetFileTime(h, &c, nil, nil)
 }
+
+// ChtimesNoFollow means to change access and modification time on a file,
+// without following symbol links.
+func ChtimesNoFollow(path string, atime time.Time, mtime time.Time) error {
```

Comment:
Perhaps we should split the exported function from the platform-specific implementation (as we did for `Chtimes()`) that was we only need to maintain the GoDoc once, and will be sure the signature is the same for both; https://github.com/moby/moby/blob/5176095455642c30642efacf6f35afc7c6dede92/pkg/system/chtimes.go#L8-L9

Reply:
Thx for the comments, I have made further code refactor and move ChtimesNoFollow into chtimes.go